src/librustc_mir/interpret/eval_context.rs

```diff
@@ -131,6 +131,10 @@ pub enum LocalValue<Tag = ()> {
 }
 
 impl<'tcx, Tag: Copy + 'static> LocalState<'tcx, Tag> {
+    /// Read the local's value or error if the local is not yet live or not live anymore.
+    ///
+    /// Note: This may only be invoked from the `Machine::access_local` hook and not from
```

Comment:
```suggestion
    /// Note: This must only be invoked from the `Machine::access_local` hook and not from
```

Reply:
I had `must` before :laughing: I think `may` is correct here.